Tests/Unit/SOGoTest.h

```diff
@@ -54,6 +54,8 @@
 
 - (BOOL) run;
 
+NSString *_stringForCharacterAtIndex(NSUInteger index, NSString *str, NSUInteger length);
```

Comment:
Try taking it out of the @implementation - so just above @implementation and below:

static NSString *SOGoTestAssertException = @"SOGoTestAssertException";

Thanks!